src/main/java/org/openelisglobal/search/service/SearchResultsController.java

```diff
@@ -0,0 +1,38 @@
+package org.openelisglobal.search.service;
+
+import java.util.List;
+import org.openelisglobal.common.provider.query.PatientSearchResults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class SearchResultsController {
+
+    @Autowired
+    private SearchResultsService searchResultsService;
+
+    @GetMapping("/search")
```

Comment:
MOve this function to the existing [PatientSearch ](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/src/main/java/org/openelisglobal/common/rest/provider/PatientSearchRestController.java)COntroller